addons/test_website/tests/test_session.py

```diff
@@ -1,9 +1,16 @@
 import odoo.tests
-from odoo.tools import mute_logger
 
 
 @odoo.tests.common.tagged('post_install', '-at_install')
 class TestWebsiteSession(odoo.tests.HttpCase):
 
     def test_01_run_test(self):
         self.start_tour('/', 'test_json_auth')
+
+    def test_02_inactive_session_lang(self):
+        session = self.authenticate(None, None)
```

Comment:
Shouldn't there be a test in base modules? Doesn't seems related to website? Any module could do this kind of code before predispatch no?

Reply:
This test is only for the override of `def _get_public_users(cls):` in the website which tries to query record before `_pre_dispatch`.